inventory/typescript/src/cell-renderers/actionsCellRenderer.ts

```diff
@@ -29,8 +26,8 @@ export class ActionsCellRenderer implements ICellRendererComp {
       rowData.status = !isPaused
         ? "paused"
         : !isOutOfStock
-          ? "active"
-          : "outOfStock";
+        ? "active"
+        : "outOfStock";
 
       // Refresh the row to reflect the changes
       api.applyTransaction({ update: [rowData] });
```

Comment:
There is a bug on this line where clicking on `Hold Selling` doesn't do anything for the typescript example

Reply:
Actually a bug in the status cell renderer (it didn't properly implement the `refresh` method)